arangod/Transaction/Methods.h

```diff
@@ -115,6 +116,9 @@ class Methods {
   /// @brief destroy the transaction
   virtual ~Methods();
 
+  Methods(Methods&&) = default;
```

Comment:
Check if this is still needed,
I know we used it in between, but this may be obsolete now

Reply:
It is needed